src/main/java/com/gmail/goosius/siegewar/events/SiegeRemoveEvent.java

```diff
@@ -0,0 +1,89 @@
+package com.gmail.goosius.siegewar.events;
+
+import com.gmail.goosius.siegewar.objects.Siege;
+import org.bukkit.Bukkit;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * This event is triggered immediately after a siege is removed (via admin command or
```

Comment:
This comment should probably be re-written to not be limited to just about the admin command.